app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,39 @@
+class TasksController < ApplicationController
+  before_action :set_task, only: [:show, :destroy]
+
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+  end
+
+  def new
+  end
+
+  def create
+    task = Task.new(task_params)
```

Comment:
You should this code, it doesn't work

Reply:
This task should not contain this code, so we can ignore this problem. We discussed it in the lesson